projects/admin/src/app/record/circulation-logs/circulation-stats/circulation-stats.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2021-2023 RERO
+  Copyright (C) 2021-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="row mb-3" *ngIf="stats">
+  <div class="col-6">
+    <span class="font-weight-bold" translate>Past year</span>
+    <ul class="list-unstyled mb-0">
+      <li>{{ 'checkout' | translate | ucfirst }}: {{ stats.total_year.checkout | default: 0 }}</li>
```

Comment:
@PascalRepond @benerken (and other PO group) : is that others circulation operations should be displayed (the API is able to return any circ op)

Reply:
No, I think the count for other operation is not useful for a professionnal.